### DISCUSSION TITLE
Test staging Stored Items 📦

### DISCUSSION THREAD

**Discussion by inikoo** [inikoo/aiku#617] [General]
Testing stored items

**Comment by KirinZero0** [comment]
![image](https://github.com/user-attachments/assets/724b7440-c41f-43a5-8549-56423d32b05c)

I think there should be an instruction for this, the rule for this form is string between 2 and 9 characters

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/af0cd37f-92c0-441e-b92c-ba8b02163cfb)
cannot open aiku stanging https://app.aiku-sandbox.uk/login

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/09d381f6-6836-4efc-9ca2-58c5f23c0cc6)
It would be better if Retina had information about the features that have been activated, so that users know whether to upgrade or not

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/72631dee-ab20-4a6e-8bbe-8146f1d395df)
![image](https://github.com/user-attachments/assets/d68ef008-7b1f-4160-8ac3-e4c5157215cc)
status items should use tags, and this list should be in retina also

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/9aed9375-f5e8-4f95-a29a-65b47f553ba2)
 show storage item dont have complete data

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/0c369b53-2402-4250-82f5-262487209105)
url : http://app.aiku.test/org/aw/fulfilments/awf/customers/3b-recycling-ltd/stored-items/aku/edit

edit storage item not work

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/2bc9fece-6689-4fc5-acc5-92e99dab399e)
![image](https://github.com/user-attachments/assets/aa0e5e68-60f1-44d4-b0e3-40de41e8fc3b)

url : http://app.aiku.test/org/aw/fulfilments/awf/customers/3b-recycling-ltd/stored-items/aku

set as damaged  & return to customer is not work

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/64f59770-fe69-4030-82ee-651880c8568e)


total pallet not match

**Comment by YudhistiraA** [comment]
https://github.com/user-attachments/assets/90b850fd-e011-4b4f-a992-0974c85bafae
You can still edit when you click back check the video

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/9765dd77-1afa-4ff7-b339-c251a2e5ade9)
http://app.aiku.test/org/aw/fulfilments/awf/customers/3b-recycling-ltd/stored-items/asdasd?tab=pallets
move to location error

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/6ae885b2-4418-462e-982d-62f539d7492d)
![image](https://github.com/user-attachments/assets/33521609-e2e7-4042-bec3-3d866c604915)
pallet alreacy booked-in but status in stored item still  in-process

**Comment by YudhistiraA** [comment]
After trying stored items, I was a bit confused by the concept. I had difficulty tracking my stored items per pallet, because there was a possibility that the stored items could move, but I couldn't track the stored items one by one.

![image](https://github.com/user-attachments/assets/2afa4902-73db-4287-8131-ac414d6c763a)

![image](https://github.com/user-attachments/assets/89b79a7a-849e-45fd-953c-8bf41cc41011)
If I set it as damaged, will all the stored items on each pallet be damaged?


I think it would be better if stored items were made separately on each pallet, it would be easier to track and modify the state and data
